step-strikers-app/step-strikers-app/Classes/RPGCharacter.swift

```diff
@@ -207,6 +207,10 @@ class RPGCharacter {
     func fight(target: inout RPGCharacter){
         let damageDealt = calculateDamage(wielderAttackModifier: self.attackModifier, wielderCurrWeapon: self.currWeapon, wielderClass: self.getCharacterClass(), target: &target)
         
+        doConsequencesOfFight(target: &target, damageDealt: damageDealt)
+    }
+    
+    func doConsequencesOfFight(target: inout RPGCharacter, damageDealt: Int) {
```

Comment:
I know this isn't a change on this pr but I didn't catch it last time - for doConsequncesOfFight() the target stuff should all be done on firebase because your phone is only locally storing your own character. If you add a TODO for me there to remind me I'll replace the target stuff you did with firebase operations